Gateway/Request/ThreeDs2DataBuilder.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
```

Comment:
what do you mean ?

Reply:
just checked the list of new files in this PR and we have:
Gateway/Request/ThreeDs2DataBuilder.php
Gateway/Request/ThreeDS2DataBuilder.php

I guess it is a mistake and one should be removed

Reply:
looks like that something is wrong with what I see in github because after checking out the branch I only see 1 file...
![Screen Shot 2019-10-29 at 14 17 03](https://user-images.githubusercontent.com/42297201/67770564-e9e8f280-fa56-11e9-87d8-d13c0630a0d4.png)